generator/internal/rust/templates/crate/src/builder.rs.mustache

```diff
@@ -76,9 +76,7 @@ pub mod {{Codec.ModuleName}} {
     }
 
     {{#Codec.Methods}}
-    {{! TODO(#1813) - fix and generate builder docs for veneers }}
-    {{^Codec.HasVeneer}}
-    /// The request builder for [{{Codec.ServiceNameToPascal}}::{{Codec.Name}}][super::super::client::{{Codec.ServiceNameToPascal}}::{{Codec.Name}}] calls.
+    /// The request builder for [{{Codec.ServiceNameToPascal}}::{{Codec.Name}}][crate::client::{{Codec.ServiceNameToPascal}}::{{Codec.Name}}] calls.
```

Comment:
Unrelated to this PR: I hate that `ServiceNameToPascal` thing, it is my fault, but I still hate it.  I think `ServiceCodec.ModuleName` is the right thing to do, assuming we had this:

https://github.com/coryan/google-cloud-rust/blob/309bcdaa47e1309f708d3209088dc64142d20389/generator/internal/api/model.go#L198-L205